extended range version.js

```diff
@@ -1,86 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-
-
-async function start() {
-
-
-
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let userMax = await ask("First, let's set the range. We'll play between 1 and a number of your choosing. Please enter the max limit now. ")
-  console.log("You entered: " + userMax)
-  let secretNumber = await ask("Now pick a number between 1 and the number you just entered - What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-
-  let min = 1
-  let max = userMax
-
-  //something about the way this is running is giving numbers way outside the specified range
-  function randNum(min, userMax) {
-    let range = (userMax - min) + 1;
-    let decimal = (Math.random() * range);
-    let int = Math.floor(decimal) + min;
-    return int;
-  }
```

Comment:
your variables `max` and `userMax` are not what you are expecting them to be. you can add debugging `console.log()` of what those variables are. and you can use `typeof()` to double check them

With them not being what you expect they are, your randNum function isn't working appropriately. 

If you want to chat more about this we can take some time to go more in depth with it